index.js

```diff
@@ -98,12 +98,16 @@ class Ky {
 			credentials: 'same-origin', // TODO: This can be removed when the spec change is implemented in all browsers. Context: https://www.chromestatus.com/feature/4539473312350208
 			retry: 3,
 			timeout: 10000,
+			throwHttpErrors: true,
 			...options
 		};
 
 		this._timeout = this._options.timeout;
 		delete this._options.timeout;
 
+		this._throwHttpErrors = this._options.throwHttpErrors;
+		delete this._options.throwHttpErrors;
+
```

Comment:
Remove these lines.

Reply:
Got it. I thought I should delete it because like timeout, it's not something we want to pass to fetch.